config.yml

```diff
@@ -840,11 +862,26 @@ nodes:
       - name: left
         type: node
         kind: pattern expression
+        comment: |
+          Represents the left side of the expression. It can be any [non-void expression](https://github.com/ruby/prism/blob/main/docs/parsing_rules.md#non-void-expression).
```

Comment:
> It can be any [non-void expression]

No, it's always a pattern expression, so that's incorrect.

Reply:
Thanks! Sorry about that -- I totally missed seeing those keys.